BazelExtensions/workspace.bzl

```diff
@@ -180,6 +183,7 @@ pod_repo_ = repository_rule(
         "enable_modules": attr.bool(default=True, mandatory=True),
         "generate_module_map": attr.bool(default=True, mandatory=True),
         "header_visibility": attr.string(),
+        "extraction_handler": attr.string(),
```

Comment:
I think a better way to do this might be to have `attr.string(values=["default", "zip", "tar"], default="default")` (so Bazel does some of the checking for us) and then expand the docs below to explicitly include the `"default"` option. This would also involve changing other defaults from `""` to `"default"`.